<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFilterDialog</class>
 <widget class="QDialog" name="AddFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="laName">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Параметры</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="2">
       <widget class="QLineEdit" name="leProcess">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="laLevel">
        <property name="text">
         <string>Уровень:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="laProcess">
        <property name="text">
         <string>Процесс:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="laText">
        <property name="text">
         <string>Текст:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="leText"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="laLog">
        <property name="text">
         <string>Журнал:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="leLog"/>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="cbLevel">
        <item>
         <property name="text">
          <string>Verbose (Подробности)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Debug (Отладка)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Info (Информация)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Warning (Предупреждение)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Error (Ошибка)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Assert (Утверждение)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fatal error (Неисправимая ошибка)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>cbLevel</tabstop>
  <tabstop>leLog</tabstop>
  <tabstop>leProcess</tabstop>
  <tabstop>leText</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddFilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddFilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
